src/apps/intermediate-list/modal/fee-details-modal.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, ReactNode } from "react";
+import Modal from "../../../core/utils/modal";
+import { useText } from "../../../core/utils/text";
+import { BasicDetailsType } from "../../../core/utils/types/basic-details-type";
+
+interface FeeDetailsModalProps {
+  faust: string;
+  material: BasicDetailsType | undefined | null;
+  children: ReactNode;
+}
+
+const FeeDetailsModal: FC<FeeDetailsModalProps> = ({ faust, children }) => {
+  const t = useText();
+
+  return (
+    <Modal
+      modalId={faust || ""}
+      classNames="modal-details"
+      closeModalAriaLabelText={t("materialDetailsCloseModalText")}
```

Comment:
```suggestion
      closeModalAriaLabelText={t("FeeDetailsModalCloseModalText")}
```

Reply:
and then add that to the strings. this would say something like: "Close fee details modal"

Reply:
Same with below "materialDetailsModalDescriptionText"